adonis-typings/i18n.ts

```diff
@@ -185,6 +185,12 @@ declare module '@ioc:Adonis/Addons/I18n' {
      */
     provideValidatorMessages: boolean
 
+    /**
+     * Add this function if you want to have control over what is returned
+     * when an identifier is missing.
+     */
+    fallBackIfNoIdentifier?: (identifier: string, locale: string) => string
```

Comment:
Just naming the method `fallback` should be fine. Because, there is no other fallback in place to cause confusion

Reply:
Renamed in latest commit